apps/microsoft-teams/app-actions/src/actions/handle-app-event.ts

```diff
@@ -24,7 +24,7 @@ export const handler = withAsyncAppActionErrorHandling(
   ): Promise<AppActionCallResponse<SendEntryActivityMessageResult[]>> => {
     const {
       cma,
-      appActionCallContext: { appInstallationId },
+      appActionCallContext: { appInstallationId, environmentId, spaceId, userId },
```

Comment:
it'll be interesting to see what value we get for `userId` here, since this action is triggered by an app backend, not a user 🤔

Reply:
Yeah good question, so we are actually passing these headers in via the frontend, so it should ideally be client-side information captured about the user if I am understanding correctly!